source/localizable/feedspec/index.el.html.md.erb

```diff
@@ -412,6 +413,21 @@ To χρώμα του προϊόντος. Το πεδίο χρώματος λει
 ~~~
 
 
+#### Weight
```

Comment:
Naming it weight will confuse those using the volume metric. I suggest either renaming it to `Shipping  Volume` or removing references to kilograms